test/TestExtensions/API.Test/InternalAPITestHook.cs

```diff
@@ -6,6 +7,18 @@ namespace API.Test
 {
     public static class InternalAPITestHook
     {
+        public static void InstallLatestPackageApi(string id, bool prerelease)
+        {
+            var dte = ServiceLocator.GetInstance<EnvDTE.DTE>();
+            var services = ServiceLocator.GetInstance<IVsPackageInstaller2>();
```

Comment:
Correct. This is operating on the bits that were just built. (if not maybe we should consider the value of the tests... :wink:)